# GRUMP-Workshop-2025
This repository contains a workbook which is intended to teach users basic R scripts which plot various data from the GRUMP dataset.

These scripts assume that you have downloaded R and R studio (https://posit.co/download/rstudio-desktop/), and that you have the data downloaded and stored in your desktop in a folder called "GRUMP-Workshop" for which a template can be downloaded from this repository. The GRUMP data can be downloaded from simons CMAP: https://simonscmap.com/catalog/datasets/GRUMP.

<img width="1512" alt="Screenshot 2025-06-15 at 12 35 44 PM" src="https://github.com/user-attachments/assets/3fd04fa4-2672-459c-b721-97124c7bc83d" />

The Global rRNA Universal Metabarcoding Plankton database (GRUMP), which consists of 1194 samples that were collected from 2003-2020 and cover extensive latitudinal and longitudinal transects, as well as depth profiles in all major ocean basins. DNA from unfractionated (> 0.2µm) seawater samples was amplified using the 515Y/926R universal three-domain rRNA gene primers, simultaneously quantifying the relative abundance of amplicon sequencing variants (ASVs) from bacteria, archaea, eukaryotic nuclear 18S, and eukaryotic plastid 16S. Thus, the ratio between taxa in one sample is directly comparable to the ratio in any other GRUMP sample, regardless of gene copy number differences. This obviates a problem in prior global studies that used size-fractionation and different rRNA gene primers for bacteria, archaea, and eukaryotes, precluding comparisons across size fractions or domains.

[Figure-1-GRUMP-Map-and-depths-V2.pdf](https://github.com/user-attachments/files/20746502/Figure-1-GRUMP-Map-and-depths-V2.pdf)
Figure 1. Map of GRUMP transects, as well as their depth resolution.


